formapp/static/js/home.js

```diff
@@ -67,17 +94,16 @@ function letterVote(letter) {
 
 function updateVotes() {
   //Make sure displayed vote matches actual vote, for example if new poll is started or user reloads
-  let res = getResponse("/getme");
   let thumb;
-  if (res.thumb == "up") thumb = 0;
-  else if (res.thumb == "wiggle") thumb = 1;
-  else if (res.thumb == "down") thumb = 2;
+  if (meRes.thumb == "up") thumb = 0;
+  else if (meRes.thumb == "wiggle") thumb = 1;
+  else if (meRes.thumb == "down") thumb = 2;
   else thumb = false;
   let letter;
-  if (res.letter == "a") letter = 0;
-  else if (res.letter == "b") letter = 1;
-  else if (res.letter == "c") letter = 2;
-  else if (res.letter == "d") letter = 3;
+  if (meRes.letter == "a") letter = 0;
+  else if (meRes.letter == "b") letter = 1;
+  else if (meRes.letter == "c") letter = 2;
+  else if (meRes.letter == "d") letter = 3;
```

Comment:
[This would be a great place for a switch statement](https://www.w3schools.com/js/js_switch.asp)